app/src/main/res/layout/login_fragment.xml

```diff
@@ -50,17 +50,41 @@
 
                 </com.google.android.material.textfield.TextInputLayout>
 
-                <TextView
-                    android:layout_width="wrap_content"
+                <LinearLayout
+                    android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:text="Here's a hint"/>
+                    android:orientation="vertical">
+
+                    <TextView
+                        android:id="@+id/display_hint_text"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:text="@{viewModel.hint}"
+                        android:layout_marginStart="15dp"
```

Comment:
we should always **try** to follow material guidelines,, basically setting margin, padding to a multiple of 8.
using 15,12,16, etc will result in inconsistent design once app grows. see [this](https://material.io/design/layout/spacing-methods.html#baseline-grid)
however I only said try to follow it.... but sometimes we have exceptional cases :)

Reply:
Yes I will try to follow the guidelines next time onwards.